flink-runtime/src/test/java/org/apache/flink/runtime/highavailability/AbstractHaServicesTest.java

```diff
@@ -66,13 +66,14 @@ void testCloseAndCleanupAllDataDeletesBlobsAfterCleaningUpHAData() throws Except
                         () -> closeOperations.offer(CloseOperations.HA_CLEANUP),
                         ignored -> {});
 
-        haServices.closeAndCleanupAllData();
+        haServices.cleanupAllData();
+        haServices.close();
```

Comment:
```suggestion
        haServices.closeWithOptionalCleanup(true);
```
This improves the test coverage (i.e. it would check the newly added default implementation).